packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemWithMenu.tsx

```diff
@@ -33,7 +33,7 @@ export class SelectedItemWithMenu extends React.Component<IPeoplePickerItemWithM
         <div className='ms-PickerItem-content'>
           <Persona
             { ...item }
-            presence={ item.presence ? item.presence : PersonaPresence.online }
+            presence={ item.presence != null ? item.presence : PersonaPresence.none }
```

Comment:
!== undefined , will result in unexpected behavior if someone sets item.presence = null. I think, I can revert the check to just item.presence as changing online to none in else case fixed the problem. so final change will be item.presence ? item.presence : PersonaPresence.none

Let me know what you think about this!